### PR TITLE
pixel window functions

### PR DESCRIPTION
**Pull request by xzackli**
This adds the `pixwin` function, tests, docstring, and a little bit in the docs.

### CODE REVIEW

**Comment by xzackli** [comment]
I might implement the full pixel weights using the same artifacts method eventually.

File: docs/src/alm.md [hunk 1]
```diff
@@ -81,10 +81,13 @@ alm2map
 ## From harmonic coefficients to the power spectrum
 
 You can use the function `alm2cl` to convert a set of $a_{\ell m}$
-coefficients into the components $C_\ell$ of the power spectrum.
+coefficients into the components $C_\ell$ of the power spectrum. 
+The pixelization also induces a transfer function, which can be obtained from
+`pixwin`.
```
Comment (ziotom78) [inline docs/src/alm.md:86]: Writing

    [`pixwin`](@ref)

should produce an hyperlink to the `pixwin` docstring in the documentation.

Comment (xzackli) [reply docs/src/alm.md]: @xzackli yes, good idea!

Comment (xzackli) [reply docs/src/alm.md]: @ziotom78 yes, good idea!

**Pull request merged**